tests/functional/test_install_extras.py

```diff
@@ -126,3 +127,22 @@ def test_install_special_extra(script):
     assert (
         "Could not find a version that satisfies the requirement missing_pkg"
     ) in result.stderr, str(result)
+
+
+@pytest.mark.skipif(sys.version_info < (3, 0),
```

Comment:
Why is this test dependant on Py3?

Reply:
`django-rest-swagger[reST]==0.3.10` requires python3. There's probably a better way to test this problem, but this works.

Reply:
I don't really like having a test that:

1. Depends on an external package that "happens" to have this issue (I describe it like that because we don't appear to have established precisely what causes the issue).
2.  Can only be tested on Python 3, even though the problem can happen on Python 2.
3. May stop working if the external package changes.

I'd rather we tested this using an artificially-constructed example that we ship with the test suite, so we can control it better.